config/routes.rb

```diff
@@ -11,4 +16,22 @@
   get '/login', to: 'sessions#new'
 
   resources :keywords, only: %i[index show create]
+
+  namespace :api do
+    namespace :v1 do
+      use_doorkeeper do
+        # Only need the tokens_controller for API V1
+        controllers tokens: 'tokens'
+
+        skip_controllers :applications, :authorizations, :token_info, :authorized_applications
+      end
+    end
+  end
+
+  namespace :api do
+    namespace :v1 do
+      resource :users, only: :create
+      resources :keywords, only: %i[index show create]
```

Comment:
Can we combine this part with the previous definition?
```ruby
  namespace :api do
    namespace :v1 do
      use_doorkeeper do
        # Only need the tokens_controller for API V1
        controllers tokens: 'tokens'

        skip_controllers :applications, :authorizations, :token_info, :authorized_applications
      end

      resource :users, only: :create
      resources :keywords, only: %i[index show create]
    end
  end
```

Reply:
I think we cannot because I want the oauth application located under `oauth/application` route instead of `/api/v1/oauth/application` as application manager is not an API